aksel.nav.no/website/pages/api/preview.ts

```diff
@@ -33,7 +33,7 @@ export default async function preview(
 ) {
   const { slug } = req.query;
 
-  if ((!slug && slug !== ":slug*") || slug instanceof Array) {
+  if ((!slug && slug !== ":slug*") || Array.isArray(slug)) {
```

Comment:
There's a logical flaw here. If `slug` is falsy, it will always also be not equal to `":slug*"`.

Reply:
true! 
<img width="579" alt="image" src="https://github.com/navikt/aksel/assets/819074/2edfc4ed-b877-4a7d-b1c7-3a668a24168a">

testet kjapt med truthy check på første condition (`fn2`)